logging/interceptor.go

```diff
@@ -194,7 +276,7 @@ func addHeaderField(ctx context.Context, fields logrus.Fields, header string) (c
 
 	fields[strings.ToLower(header)] = field
 
-	return metadata.AppendToOutgoingContext(ctx, header, field), nil
+	return ctx, nil
```

Comment:
If we are not changing anything in context, can we remove it from return statement ?

Reply:
That's a good point